packages/storyshots/src/index.js

```diff
@@ -53,11 +53,18 @@ export default function testStorySnapshots(options = {}) {
   const suit = options.suit || 'Storyshots';
   const stories = storybook.getStorybook();
 
+  // Added not to break existing storyshots configs (can be removed in a future major release)
+  options.storyNameRegex = options.storyNameRegex || options.storyRegex;
```

Comment:
Awesome. We are planning a major release soon! @ndelangen we should set up a place to track breaking changes